omnibus/files/private-chef-cookbooks/private-chef/recipes/bookshelf.rb

```diff
@@ -48,12 +48,18 @@
 
 bookshelf_config = File.join(bookshelf_dir, "sys.config")
 
+bookshelf_params = node['private_chef']['bookshelf'].to_hash.dup # is dup implicit in #to_hash?
+bookshelf_params['postgresql'] = node['private_chef']['postgresql'].dup
+['sql_user', 'sql_password', 'sql_db_timeout'].each do |v|
+  bookshelf_params['postgresql'][v] = node['private_chef']['opscode-erchef'][v]
+end
```

Comment:
This was left over from when we didn't have a separate db user for bookshelf.  Will remove.